src/components/time-picker/time-picker.st.css

```diff
@@ -0,0 +1,91 @@
+@namespace 'time-picker';
+
+:import {
+    -st-from: "../stepper/stepper.st.css";
+    -st-default: Stepper;
+}
+
+:import {
+    -st-from: "../../style/default-theme/variables.st.css";
+    -st-named:
+        BK2,
+        BK3,
+        PR2,
+        PR3;
+}
+:vars {
+    height: 36px;
+    width: 120px;
+}
+.root {
+    -st-states: focus, disabled;
+    font-family: arial;
+    font-size: 16px;
+    position: relative;
+    border: 1px solid value(BK3);
+    height: value(height);
+    width: value(width);
+    display: inline-flex;
+}
+
+.root:not(:disabled):hover {
+    border-color: value(PR3);
+}
+.root:focus {
+    border-color: value(PR2);
+}
+.root:disabled {
+    color: value(BK2);
+}
+
+.input {
+    font: inherit;
+    border: 0;
+    height: 24px;
+    color: inherit;
+    width: 100%;
+    height: 100%;
+    padding: 6px;
+    background: none;
+    user-select: none;
+}
+.input::placeholder {
+    color: value(BK2);
+}
+.input::selection {
+    color: white;
+    background: value(PR2);
+    padding: 2px;
+}
+.input:hover,
+.input:focus {
+    border: 0;
+    outline: 0;
+}
+.native-input {
+    position: absolute;
+    opacity: 0;
+    padding: 0;
+    margin: 0;
+    border: 0;
+    width: 0;
+    height: 0;
+}
+.native-input:focus {
+    border: 0;
+}
+
+.stepper {
+    -st-extends: Stepper;
+    height: 100%;
+    transition: opacity 100ms ease;
+    opacity: 0;
+}
+.root:focus .stepper,
+.root:hover .stepper {
+    opacity: 1;
+}
+.stepper::control:disabled {
```

Comment:
I haven't seen it. just looked at code.
Maybe there is some styling product issue then?